lighthouse-cli/test/fixtures/perf/delayed-element.js

```diff
@@ -6,23 +6,6 @@
 
 /* eslint-disable */
 
-/** Create long tasks on the main thread. */
-function stall(ms) {
```

Comment:
I think you need to keep this for the end `stall(800)` (probably the smoke failure?)